core/src/test/java/org/testcontainers/junit/GenericContainerRuleTest.java

```diff
@@ -320,6 +323,68 @@ public void createContainerCmdHookTest() {
         }
     }
 
+    @Test
+    public void copyToContainerTest() throws Exception {
+        //compare purpose
+        File outputFile = new File("src/test/resources/copy-from/test_copy_to_container.txt");
```

Comment:
output file should not be placed in `src/test`, use a temporary directory instead